templates/index.html

```diff
@@ -49,28 +49,35 @@ <h2 class="p-text--small-caps">What is MicroK8s</h2>
 </section>
 
 <section class="p-section">
-  <div class="row--25-75 p-section">
+  <div class="row--25-75 p-section--shallow">
     <hr class="p-rule">
     <div class="col">
       <h2 class="p-text--small-caps">Why MicroK8s</h2>
     </div>
     <div class="col">
       <p class="p-heading--2">The best Kubernetes experience for developers, DevOps, cloud and edge</p>
-    </div>    
+    </div>
   </div>
   <div class="row">
-    <hr class="p-rule col-start-large-4 col-9">
-    <div class="col-start-large-4 col-3">
-      <h3 class="p-heading--5">For Developers</h3>
-      <p>MicroK8s is the easiest and fastest way to get Kubernetes up and running. Experiment with the latest upstream features and toggle services on and off. Seamlessly move your work from dev to production.</p>
-    </div>
-    <div class="col-3">
-      <h3 class="p-heading--5">For DevOps</h3>
-      <p>With self-healing high availability, transactional OTA updates and secure sandboxed kubelet environments, MicroK8s is the go-to platform for mission-critical workloads. Quickly spin nodes up in your CI/CD and reduce your production maintenance costs.</p>
-    </div>
-    <div class="col-3">
-      <h3 class="p-heading--5">For software vendors</h3>
-      <p>Leverage the simplicity, robustness and security of MicroK8s as a full embedded Kubernetes platform. Build containerised solutions with a system that doesn't require your attention. Focus on your customers, not the infrastructure.</p>
+    <div class="col-9 col-start-large-4 col-medium-6">
+      <div class="p-section--shallow">
+        <img src="https://assets.ubuntu.com/v1/6bb2f8e7-stocksy-woman-working-ar2-1.jpg" alt="">
```

Comment:
Can you use image template whenever possible? This should improve page loading times especially for large images

```

          {{ image (
            url="https://assets.ubuntu.com/v1/6bb2f8e7-stocksy-woman-working-ar2-1.jpg",
            alt="",
            width="2048",
            height="1024",
            hi_def=True,
            loading="auto|lazy"
            ) | safe
          }}
```

You can generate one clicking 
<img width="659" alt="Screenshot 2023-11-27 at 14 48 48" src="https://github.com/canonical/microk8s.io/assets/14939793/ced9180e-a67f-4058-901f-00ccdd7815c6">